sumpy/tools.py

```diff
@@ -976,19 +981,198 @@ def wait(self):
         return self.native_event.wait()
 
 
-def get_opencl_fft_app(queue, shape, dtype):
-    """Setup an object for out-of-place FFT on with given shape and dtype
-    on given queue. Only supports in-order queues.
-    """
+def loopy_fft(shape, inverse, complex_dtype, index_dtype=None,
```

Comment:
Could you add a unit test for this?

Reply:
Done in 0e5b4ced0dcc681ccea235135d44dbf929d4e6ee